### PR TITLE
Adds various carbon and human emotes

### PR DESCRIPTION
**Pull request by ghost**
Adds some emotes I either find myself upset when I can't do them or think are worthwhile and should have been in before.

### CODE REVIEW

**Comment by Xhuis** [comment]
*gargle *tongue
you fool

File: code/modules/mob/living/carbon/human/emote.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /mob/living/carbon/human/emote(act,m_type=1,message = null)
 	var/param = null
-
+//*grimace "<B>[src]</B>\'s" *rolleye *huff *blanch *snigger *smirk *cutthroat *gulp *tongue *grit *flounder *flush *slump *stomp *gargle *scowl
```
Comment (RemieRichards) [inline code/modules/mob/living/carbon/human/emote.dm]: remove comment.

**Comment by Steelpoint** [comment]
> flushes

What have you done.

**Comment by PKPenguin321** [comment]
>gulp
>tongue
>flush
>gargle

welcome to erp verbs 2.0

**Comment by Ricotez** [comment]
might as well have added *deepthroat at this rate

**Comment by Shapsy** [comment]
You should also add *scoff

**Comment by CosmicScientist** [comment]
@Shapsy would that show as:
**Humie** scoffs.
?

Because if so, sure I can add it in.

File: code/modules/mob/living/carbon/human/emote.dm [hunk 2]
```diff
@@ -100,6 +103,21 @@
...
+				var/mob/M = null
+				m_type = 1
+				if (param)
+					for (var/mob/A in view(src))
+						if (param == A.name)
+							M = A
+							break
+					if (M != src && M != null)
+						var/genderType = gender == MALE ? "his" : "her"
+						message = "<B>[src]</B> points [genderType] finger at [M] and then draws a line across [genderType] throat."
```
Comment (phil235) [inline code/modules/mob/living/carbon/human/emote.dm]: use \his just like below, instead of GenderType

Comment (CosmicScientist) [inline code/modules/mob/living/carbon/human/emote.dm]: I'll test it to see if it does work but the reasoning for that was being told the second \his would pick on [M]'s gender.

**Comment by Shapsy** [comment]
ye

File: code/modules/mob/living/carbon/human/emote.dm (hunk 2 above)
Comment (CosmicScientist) [inline code/modules/mob/living/carbon/human/emote.dm]: Testing proved it turns \his into the gender/non-gender of [M], unless I write the phrase differently, e.g. "[src] draws a line across \his throat and points at [M]", the previous line is needed.

**Comment by CosmicScientist** [comment]
Flush is now a little different:
**Sickly patient (405)** flushes with colour.
**Sickly patient (405)** says "Thanks doc, I feel much better now."

Since there's already blush I made it a bit more general to be an opposite to blanch.

File: code/modules/mob/living/carbon/human/emote.dm (hunk 2 above)
Comment (phil235) [inline code/modules/mob/living/carbon/human/emote.dm]: my bad then, good catch

**Comment by Miauw62** [comment]
>adding more emotes to the disgusting heap of shitcode that is emotes

i really need to do something about that, someday.

also more like
**Medbay stutterwhore #5897236** flushes with colour.
**Medbay stutterwhore #5897236** says, "O-oh~"

**Comment by CosmicScientist** [comment]
@Miauw62 they can already meme with *blush, *me and the custom command. Remove those or change the player base.

**Comment by Miauw62** [comment]
i was making a joke :v

**Comment by CosmicScientist** [comment]
Sorry, it's hard to tell what's a joke and what's not sometimes on here. ='(

**Comment by Xhuis** [comment]
I still think that some of these emotes are way too erotic

**Comment by CosmicScientist** [comment]
Which ones @Xhuis? I can solve it by adding *doubledongs so the rest seem PG13 by comparison.

File: code/modules/mob/living/carbon/human/emote.dm (hunk 2 above)
Comment (CosmicScientist) [inline code/modules/mob/living/carbon/human/emote.dm]: Not my catch, it was Remie's.

**Comment by CosmicScientist** [comment]
This isn't meaningful and bloating this bit of code further isn't good.

I hope datum emotes become a thing.

**Pull request closed without merging**